apps/weed-map/src/routes/strains/[strain]/index.tsx

```diff
@@ -0,0 +1,234 @@
+import { component$, useStore, useStyles$, useStylesScoped$ } from '@builder.io/qwik';
+import type { DocumentHead } from '@builder.io/qwik-city';
+import { Link, routeLoader$ } from '@builder.io/qwik-city';
+import { getSupabaseClient } from '@api/client';
+import StrainStyles from './strain.css?inline';
+import { Icon } from '@ui/icon';
+
+export const head: DocumentHead = {
+  title: 'Weed Map - Strain Detail',
+  meta: [
+    {
+      name: 'description',
+      content: 'Store detail...',
+    },
+  ],
+};
+
+const mockStrain = {
+  id: 1,
+  slug: 'strain1',
+  name: 'Strain name 2',
+  img_url: 'https://images.leafly.com/flower-images/gg-4.jpg',
+  rating: {
+    value: 4.6,
+    ratings_count: 5142,
+    reviews_count: 1022,
+    likes_count: 82265,
+  },
+  created_at: '2023-07-09T12:54:01.829421+00:00',
+  updated_at: '2023-07-09T12:54:01.829421+00:00',
+  genetic: 'Hybrid',
+  purity: { min: 90, max: 100 },
+  thc: { min: 20, max: 20 },
+  cbd: { min: 0, max: 0 },
+  price: { min: 10, max: 15, currency: 'USD' },
+  parents: ['strain2'],
+  aliases: ['Gorilla Glue', 'Sticky Glue'],
+  description:
+    'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Etiam bibendum elit eget erat. Nullam eget nisl. Duis condimentum augue id magna semper rutrum. Morbi leo mi, nonummy eget tristique non, rhoncus non leo. Mauris metus. In enim a arcu imperdiet malesuada. Integer lacinia. Mauris elementum mauris vitae tortor. Aliquam in lorem sit amet leo accumsan lacinia. Fusce tellus. Etiam commodo dui eget wisi. Nulla pulvinar eleifend sem. Nullam justo enim, consectetuer nec, ullamcorper ac, vestibulum in, elit. Nulla quis diam. Praesent dapibus. Quisque porta. Aliquam in lorem sit amet leo accumsan lacinia. Praesent vitae arcu tempor neque lacinia pretium.',
+  feelings: [
+    { name: 'relaxed', evidence: 30 },
+    { name: 'sleepy', evidence: 30 },
+    { name: 'hungry', evidence: 30 },
+  ],
+  negatives: [
+    { name: 'dry mouth xyzas asd asd asd  asd asd sa dsa d', evidence: 30 },
+    { name: 'paranoid', evidence: 30 },
+    { name: 'dry eyes', evidence: 30 },
+  ],
+  helps: [
+    { name: 'stress', evidence: 30 },
+    { name: 'anxiety', evidence: 30 },
+    { name: 'pain', evidence: 30 },
+  ],
+  flavors: [
+    { name: 'pungent', evidence: 30 },
+    { name: 'pine', evidence: 30 },
+    { name: 'earthy', evidence: 30 },
+  ],
+  terpene_dominant: 'caryophyllene',
+  terpene_other: ['myrcene', 'limonene'],
+  likes: true,
+};
+
+export const useStrain = routeLoader$(async (req) => {
+  const { strain } = req.params;
+  const client = getSupabaseClient(req);
+  const { data } = await client.from('strains').select('*').eq('slug', strain).single();
```

Comment:
Použil bych v ceste slug, misto strain, at je to konzistentni a jasne na prvni pohled. Tzn `routes/strains/[slug]/index.tx`